### PR TITLE
Errata

### PR DESCRIPTION
**Pull request by thzinc**
I was working through your very helpful guide and found a few errata. I didn't see a contributor's guide. Please let me know if you want any changes to be made to this.

Thanks for your walkthrough!

### CODE REVIEW

**Comment by at-manos** [comment]
Appreciate the corrections, all looks good to me! 

Contributing guide is planed, not exactly sure what to put in there though. If you have any ideas I'd be open to hearing them, relatively new to all this.

**Comment by thzinc** [comment]
@at-manos, I usually start from [a set of templates I created](https://github.com/syncromatics/Syncromatics.Engineering.DX.Common) and then customize the `CONTRIBUTING.md` as needed. In the case of this repo, some of the things that come to my mind in no particular order are:

* Is there a specific flavor of Markdown that's preferred?
* Is there a particular tone or author's voice to use? (e.g., something like the well-defined [Mailchimp style guide](https://styleguide.mailchimp.com/))
* Do you prefer to have contributors create issues, then associate PRs to the issues, OR are you okay with a direct PR like I've shared here?
* Are there rules for commit messages? (tools like semantic-release, GitVersion, gogitver, etc. use tags in commit messages for versioning)

Basically, these are the questions I had when I put this PR together. I also maintain a few open source repos and have an idea of what I prefer, so I try to be pretty chill with my contributions and make sure I'm not creating a bunch of work for other maintainers.

**Comment by at-manos** [comment]
Thanks for all the great info @thzinc ! Really appreciate it. I'll start putting together a `CONTRIBUTING.md`. 

I don't know that I prefer either issues+PR or just PR in particular. Is there any reason past personal preference to use them?  I've always just seen issues as a way for someone to introduce a problem that they don't have the time to resolve/make a PR for. If it's just personal preference, what would your recommendation for a documentation project like this be? 

I looked into semantic-release and will probably adopt it here, with the angular commit format.

Again thanks for all the info, really helps out!

**Comment by thzinc** [comment]
@at-manos, the issue+PR workflow is usually preferred for bigger or higher-velocity projects. (perhaps like the `flipperzero-firmware` repo itself). I think the direct PR workflow is probably fine for this project at this time, and if you pair it with [a `PULL_REQUEST_TEMPLATE.md`](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) to nudge contributors to explain their changes, you'll probably have a decent time as a maintainer.

**Comment by at-manos** [comment]
I agree, but I'm a little conflicted since I don't want someone to spot a mistake and not report it because they don't want to go through all the effort of cloning, editing, opening a PR, etc. I'll look into making the PR template.
